src/components/gds-button/_index.scss

```diff
@@ -46,6 +46,9 @@
 @mixin text-button {
   color: inherit;
   background-color: inherit;
+  font-weight: var(--button-text-button-font-weight);
```

Comment:
This naming is not ideal but not sure how to support these kinds of variations

Reply:
Yeah, that is tricky. Maybe the best way would be to have a separate gds-text-button? Another way would be just to scope the and redefine the --button-font-weight variable in the destination project?

Reply:
`gds-text-button` sounds like a good option, i bet they can look vastly different on some projects.

Reply:
Yeah, and they might also behave differently